shared/router-v2/router.native.js

```diff
@@ -153,7 +153,20 @@ const LoggedInStackNavigator = createStackNavigator(
 const LoggedOutStackNavigator = createStackNavigator(
   {...Shim.shim(loggedOutRoutes)},
   {
-    defaultNavigationOptions,
+    defaultNavigationOptions: {
```

Comment:
make the logged out stack have cancel buttons so signup / provision work correctly